openmc/lattice.py

```diff
@@ -1163,7 +1163,7 @@ def find_element(self, point):
             iz = 1
         else:
             z = point[2] - self.center[2]
-            iz = floor(z/self.pitch[1] + 0.5*self.num_axial)
+            iz = floor(z/self.pitch[1] + 0.5 * self.num_axial)
```

Comment:
PEP8 says "[If operators with different priorities are used, consider adding whitespace around the operators with the lowest priority(ies).](https://www.python.org/dev/peps/pep-0008/#other-recommendations)" and they give some examples with mixed multiplication and addition. This change seems to go against what PEP8 says.

Reply:
These blocks were flagged by PullRequest tools in a scan for Potential High Cognitive Load. In these cases, PullRequest engineers will look for opportunities to break up operations or make code more readable. The consensus among the reviewing team was that additional spaces, despite PEP8's standard, made the flow "slightly more readable."

This is strictly preference, and not a strong one, so I'll go ahead and revert these 👍 
******
<p align="left">
    <a href="https://www.pullrequest.com/">
        <img width="15%" src="https://static.pullrequest.com/images/dashboard/pullrequest-logo.svg">
    </a>
</p>